src/algebraic_geometry/structure_sheaf.lean

```diff
@@ -857,9 +857,10 @@ instance to_basic_open_epi (r : R) : epi (to_open R (basic_open r)) :=
   swap 5, exact is_localization.to_basic_open R r, exact h }⟩
 
 @[elementwise] lemma to_global_factors : to_open R ⊤ =
-  (CommRing.of_hom (algebra_map R (localization.away (1 : R)))) ≫ (to_basic_open R (1 : R)) ≫
+  CommRing.of_hom (algebra_map R (localization.away (1 : R))) ≫ to_basic_open R (1 : R) ≫
```

Comment:
This change doesn't look related to the PR; can you either remove it, or add a sentence to the description?

Reply:
done!